addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -138,6 +138,9 @@
             </t>
         </div>
         <button t-if="event.event_registrations_open" type="button" data-bs-toggle="modal" data-bs-target="#modal_ticket_registration" class="btn btn-primary w-100">Register</button>
+        <div t-if="registration_error_code == 'insufficient_seats'" class="alert alert-danger my-3" role="alert">
+            <span>Registration failed! These tickets are not available anymore.</span>
```

Comment:
I see a lot of places where the alert message is directly set under the div so I think the `span` could be removed here :thinking: (to be verified)

Also I'm not sure updating the xml structure is recommended in stable but as it brings an important error feedback to the users I'd say let's keep it :slightly_smiling_face:

Reply:
Just checked and yep, the span is useless, I just removed it ^^